docs/specification/current.md

```diff
@@ -220,7 +220,7 @@ A CDS Client MAY choose to honor some or all of the desired prefetch templates,
 - The CDS Client MAY compute an efficient set of prefetch templates from multiple CDS Services, thereby reducing the number of calls to a minimum
 - The CDS Client MAY satisfy some of the desired prefetched templates via some internal service or even its own FHIR server.
 
-The CDS Client SHALL deny access to the requested resource if the user in context is not authorized to access the resource.
+The CDS Client SHALL only provide access to the requested resource if it is within the user's authorized scope.
```

Comment:
I'd rather say...

```suggestion
The CDS Client SHALL only provide access to the resources that the user is authorized to see
```

... but the underlying Jira issue specifies this language. (In which case, why review? ;-))